src/components/navigation/Logo.tsx

```diff
@@ -27,11 +31,15 @@ export default function Logo(props) {
 }
 
 const Image = styled.img`
-  padding-left: 6px;
+  padding-left: ${logoPaddingLeft}px;
 `
 
+const sublineH2PaddingLeft =
+  logoPaddingLeft +
+  (logoWordmarkPaddingLeft * logoTargetWidth) / logoOriginalWidth
```

Comment:
probably worth a comment what you are doing here, like "adjusting padding to log size"?